api/pkg/s3/listbuckets.go

```diff
@@ -45,7 +45,13 @@ func parseListBuckets(list *s3.ListBucketsResponse) []byte {
 				versionOpts.Status = utils.VersioningEnabled
 			}
 		}
-		bucket := model.Bucket{Name: value.Name, CreateTime: ctime, LocationConstraint: value.DefaultLocation, VersionOpts: versionOpts}
```

Comment:
Is it better to provide a seperate API to get bucket SSE configuration, than return it in the List Bucket API?

Reply:
hmm...this is used to list the buckets, SSE is one attribute of a bucket. So if we have 100 buckets, one call will return 100 bucket info, including SSE, instead of 200 calls, for example, to list the SSE column on dashboard. If you feel a separate API is ALSO needed, do revert here, i will add it

Reply:
Ehmm.......I'm just thinking... if we need to want to be compatible with aws s3, as it's the de facto standard, then we need to provide a separate API.

Reply:
highly debatable...not convinced of the rationale in doing this